### ISSUE TITLE
[info] a shader that displays a png image.

### ISSUE THREAD

**Issue by 0wwafa** [libretro/slang-shaders#649]
I wish to have a shader that instead of the "source" displays a PNG image.

I have no idea how to do so.

**Comment by 0wwafa** [comment]
> You can just load the image directly in RetroArch via 'load content'. Many of the shader authors do this for testing specific scenes/images.

yep I know that but I am not using retroarch but DENISE emulator.

**Comment by hizzlekizzle** [comment]
in that case, you can use the imgborder shader. Just don't have an alpha cutout and it'll cover the whole image.

**Comment by hizzlekizzle** [comment]
works fine here.

imgborder looks for a texture named 'pvm.png' in the borders/textures directory. You can either edit imgborder.slangp to point to some other image or you can rename your image to 'pvm.png' and overwrite the existing one.
